src/gui/DatabaseTabWidget.cpp

```diff
@@ -112,126 +97,116 @@ Database* DatabaseTabWidget::execNewDatabaseWizard()
             tr("The created database has no key or KDF, refusing to save it.\n"
                "This is definitely a bug, please report it to the developers."),
             QMessageBox::Ok, QMessageBox::Ok);
-        return nullptr;
+        return {};
```

Comment:
Shouldn't this only be set internally by the database class itself?

Reply:
Assuming you mean

    db->setInitialized(true);

Not by the DB, because the DB doesn't really know what the user is doing with it. But it should be set in the database wizard. I'll move it there.

Reply:
ok good idea, yes right line